### PR TITLE
PEP-257 bodhi.server.config.

### PR DESCRIPTION
**Pull request by bowlofeggs**
Signed-off-by: Randy Barlow <randy@electronsweatshop.com>

### CODE REVIEW

File: bodhi/server/config.py [hunk 1]
```diff
@@ -563,21 +586,25 @@ class BodhiConfig(dict):
     }
 
     def __getitem__(self, *args, **kw):
+        """Ensure the config is loaded, and then call the superclass __getiten__."""
```
Comment (stickster) [inline bodhi/server/config.py]: getitem?

**Review by stickster — CHANGES_REQUESTED** [review]
Just a simple typo fix

File: bodhi/server/config.py (hunk 1 above)
Comment (bowlofeggs) [reply bodhi/server/config.py]: @stickster I've fixed the typo in a new commit I've pushed - look good to you?

**Comment by bowlofeggs** [comment]
jenkies test

**Review by stickster — APPROVED** [review]

**Pull request merged**